arangod/IResearch/IResearchFilterFactory.cpp

```diff
@@ -1568,7 +1570,6 @@ arangodb::Result fromFuncAnalyzer(irs::boolean_filter* filter, QueryContext cons
     shortName = analyzerIdValue;
 
     if (ctx.trx) {
```

Comment:
If now we does not expect ctx.trx to be nullptr, we could safely remove else branch and if condition operator. Same way like in aqlFnTokens function